apps/gbfs/lib/gbfs/controllers/index_controller.ex

```diff
@@ -0,0 +1,20 @@
+defmodule GBFS.IndexController do
+    use GBFS, :controller
+
+    plug :put_view, GBFS.FeedView
+
+    def index(%{assigns: %{networks: networks}} = conn, _params) do
+        conn
+        |> assign(:data, networks |> Enum.map(& %{
```

Comment:
I think it’s more readable if you create a private function `defp toto(name)`  to pass it to map function